src/CashierFake.php

```diff
@@ -0,0 +1,247 @@
+    /**
+     * Assert if the SubscriptionUpdated event was dispatched based on a truth-test callback.
+     *
+     * @param  callable|int|null  $callback
+     * @return void
+     */
+    public static function assertSubscriptionUpdated($callback = null)
+    {
+        Event::assertDispatched(SubscriptionUpdated::class, $callback);
+    }
+
+    /**
+     * Assert if the SubscriptionCancelled event was dispatched based on a truth-test callback.
+     *
+     * @param  callable|int|null  $callback
+     * @return void
+     */
+    public static function assertSubscriptionCancelled($callback = null)
+    {
+        Event::assertDispatched(SubscriptionCancelled::class, $callback);
+    }
+
+    /**
+     * Format the given path into a full API url.
+     *
+     * @param  string  $path
+     * @return string
+     */
+    public static function getFormattedVendorUrl(string $path): string
+    {
+        return Cashier::vendorsUrl()
+               .'/api/2.0'
+               .Str::start($path, '/');
+    }
+
+    /**
+     * Returns the default endpoints and their faked responses.
+     *
+     * @return array
+     */
+    protected function defaultVendorEndpoints()
+    {
+        return [
+            'payment/refund' => [
+                'success' => true,
+                'response' => [
+                    'refund_request_id' => 12345,
+                ],
+            ],
+
+            'subscription/users' => function () {
+                return [
+                    'success' => true,
+                    'response' => [
+                        [
+                            'subscription_id' => 3423423,
+                            'user_email' => 'john@example.com',
+                            'payment_information' => static::$paymentInformation,
+                            'last_payment' => [
+                                'amount' => 0.00,
+                                'currency' => 'EUR',
+                                'date' => '',
+                            ],
+                        ],
+                    ],
+                ];
+            },
+
+            'subscription/modifiers' => [
+                'success' => true,
+                'response' => [
+                    [
+                        'modifier_id' => 6789,
+                        'sucscription_id' => 3423423,
+                        'amount' => 15.00,
+                        'currency' => 'EUR',
+                        'is_recurring' => false,
+                        'description' => 'This is a test modifier',
+                    ],
+                ],
+            ],
+
+            'subscription/modifiers/create' => [
+                'success' => true,
+                'response' => [
+                    'subscription_id' => 3423423,
+                    'modifier_id' => 6789,
+                ],
+            ],
+
+            'subscription/modifiers/delete' => [
+                'success' => true,
+            ],
+        ];
+    }
```

Comment:
I don't think we should provide default responses and always let people indicate what should be returned.

Reply:
Anyways...I moved this to the constructor in CashierFake if you want to review...

Reply:
Yes that's okay.